include/miral/miral/runner.h

```diff
@@ -90,6 +105,14 @@ class MirRunner
     MirRunner& operator=(MirRunner const&) = delete;
     struct Self;
     std::unique_ptr<Self> const self;
+
+    struct SignalInfo
+    {
+        std::initializer_list<int> signals;
+        std::function<void(int)> const& handler;
+    };
```

Comment:
This struct is not used in the public interface: it should not be declared in the public interface.